<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envire Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Graph"/>
   <addaction name="actionSave_Graph"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Frame"/>
   <addaction name="actionRemove_Frame"/>
   <addaction name="actionAdd_Item"/>
  </widget>
  <action name="actionAdd_Frame">
   <property name="text">
    <string>Add Frame</string>
   </property>
   <property name="toolTip">
    <string>Add a new frame to the envire graph</string>
   </property>
  </action>
  <action name="actionRemove_Frame">
   <property name="text">
    <string>Remove Frame</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected frame</string>
   </property>
  </action>
  <action name="actionLoad_Graph">
   <property name="text">
    <string>Load Graph</string>
   </property>
  </action>
  <action name="actionSave_Graph">
   <property name="text">
    <string>Save Graph</string>
   </property>
  </action>
  <action name="actionAdd_Item">
   <property name="text">
    <string>Add Item</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
